packages/keystone/rateLimiting/plugin.js

```diff
@@ -246,8 +248,12 @@ class ApolloRateLimitingPlugin {
                 })
 
                 if (incrValue > allowedQuota) {
-                    // TODO(INFRA-760): Throw error instead here
-                    return
+                    throw new GQLError({
+                        code: TOO_MANY_REQUESTS,
+                        type: ERROR_TYPE,
+                        message: 'You\'ve made too many requests recently, try again later',
+                        messageForUser: `api.global.rateLimit.${ERROR_TYPE}`,
```

Comment:
why not just hardcode it like we usually do?
```suggestion
                        messageForUser: `api.global.rateLimit.RATE_LIMIT_EXCEEDED`,
```

Reply:
idk 🤷‍♂️